warp/yul/BuiltinHandler.py

```diff
@@ -431,6 +431,17 @@ def __init__(self, function_args: str):
     def required_imports(self):
         return {"evm.sha3": {"sha"}}
 
+# ============ Call Data ============
+class Caller(BuiltinHandler):
+    def __init__(self, function_args: str):
+        super().__init__(
+            module="",
```

Comment:
I think you can specify module here as "evm.calls" and then you won't have to override `required_imports`

Reply:
Yea that seems like a better way of doing it, I have updated the PR.